docs/semgrep-appsec-platform/jira.md

```diff
@@ -94,56 +93,68 @@ If you opt for a drop-down or a checkbox issue type field, verify that:
 
 ### Code
 
-To create tickets for one or more Code findings, you can use the triage button:
+You can create tickets for Code findings using the **Triage** button on the:
+
+* [**Findings**](https://semgrep.dev/orgs/-/findings) page
+* Individual finding's **Details** page
 
-* On the **Findings** page
-* On an individual finding's **Details** page.
+To create tickets:
 
-When working on the **Findings** page, you can select and create tickets for individual findings or all findings for a given rule.
+1. If you're on the [**Findings**](https://semgrep.dev/orgs/-/findings) page, select the findings for which you want tickets created; you can select and create tickets for individual findings or all findings for a given rule. Otherwise, proceed to step two.
```

Comment:
I wonder, is "step two" something Vale enforced? Because for an exact match in the reader's POV for the number to read as "Proceed to step **N**" (i.e. the arabic number, so "Proceed to step 5" or" Proceed to step 2") so that they can match the arabic "2" to the 2 at the start of the next step.

WDYT?

Reply:
I think the Arabic 2 make sense (I don't think value enforces this; I think this was just me unable to decide which option when writing and then just picking one)